.github/workflows/auto-merge.yml

```
@@ -7,6 +7,7 @@ on:
 permissions:
   pull-requests: write
   contents: write
+  workflows: write
```

actionlint (1.7.12)

[error] unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)